### PR TITLE
Paper ui setup and integration - Tested w/ Master

### PR DESCRIPTION
**Pull request by SethDonohue**
We still have to get the backend ML labels lined up with our json/ text files/labels on the front end, but this is a huge change to styling and UI/UX so i'd like to get it merged to Master.

Have already merged master and tested this all.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,34 +1,109 @@
...
+        tabBarIcon: tabBarIcon('home'),
+        // tabBarColor: 'green',
+      })
+    },
+    Camera: {
+      screen: CameraStack,
+      navigationOptions: ({ navigation }) => {
+        // Seth - Hide Tab bar when using the camera.
+        let tabBarVisible = true;
+        if(navigation.state.index === 1) tabBarVisible = false;
```
Comment (AndyCErnst) [inline App.js]: would be simpler as `const tabBarVisible = navigation.state.index !== 1`, although I'm a bit confused as to what index is here

File: infoTree/getJSX.js [hunk 2]
```diff
@@ -14,19 +14,21 @@ function getJSX(node) {
...
+    return children.map((child, index) => {
       switch (typeof(child)) {
         case "string":
-          return <Intermediate type = {nodeType} text = {child}></Intermediate>;
+          return <Intermediate key={index} type={nodeType} text={child}></Intermediate>;
         case "object":
           //for links
           if (child.type == "link"){
-            return <Link>{child}</Link>;
+            console.log('getJSX.js Object case: ', child);
```
Comment (AndyCErnst) [inline infoTree/getJSX.js]: unless this is a WIP, we should remove this and other logs

File: src/assets/icons.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  </Svg>
+);
+
+// export default tabBarIcon;
```
Comment (AndyCErnst) [inline src/assets/icons.js]: delete unused code

File: src/assets/icons.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
...05 20.4609375,29.1718756 20.4609375,29.046875 C20.4609375,27.7552019 20.674477,26.6927125 21.1015625,25.859375 C21.528648,25.0260375 22.3828061,24.0885469 23.6640625,23.046875 C24.9453189,22.0052031 25.7109362,21.3229183 25.9609375,21 C26.3463561,20.4895808 26.5390625,19.9270864 26.5390625,19.3125 C26.5390625,18.4583291 26.1979201,17.7265655 25.515625,17.1171875 C24.8333299,16.5078095 23.9140683,16.203125 22.7578125,16.203125 C21.6432236,16.203125 20.7109412,16.5208302 19.9609375,17.15625 C19.2109338,17.7916698 18.6953139,18.7604102 18.4140625,20.0625 L14.3828125,19.5625 C14.4973964,17.6979073 15.2916593,16.1145898 16.765625,14.8125 C18.2395907,13.5104102 20.1744672,12.859375 22.5703125,12.859375 C25.0911584,12.859375 27.0963467,13.5182226 28.5859375,14.8359375 C30.0755283,16.1536524 30.8203125,17.6874913 30.8203125,19.4375 C30.8203125,20.4062548 30.5468777,21.3229123 30,22.1875 C29.4531223,23.0520877 28.2838631,24.2291592 26.4921875,25.71875 C25.5650995,26.4895872 24.9895845,27.1093727 24.765625,27.578125 C24.5416655,28.0468773 24.4401041,28.8854106 24.4609375,30.09375 Z M20.4765625,36 L20.4765625,31.609375 L24.8671875,31.609375 L24.8671875,36 L20.4765625,36 Z" id="path-1"></Path>
```
Comment (AndyCErnst) [inline src/assets/icons.js:24]: you typically see these in separate files so they are easier to edit/replace/read but I suppose we could do it inlne

File: src/assets/styles.js [hunk 5]
```diff
@@ -9,48 +9,96 @@ export const largeFontSize = 16;
...
+    fontSize: standardFontSize,
+    color: colors.black,
+  },
+  Bold: {
+    margin: standardMargin,
+    lineHeight: largeLineheight,
+    fontSize: standardFontSize,
+    color: colors.black,
+    fontWeight: 'bold'
+  },
```
Comment (AndyCErnst) [inline src/assets/styles.js:103]: really happy to see this refactor

File: src/components/Intermediate.js [hunk 6]
```diff
@@ -1,47 +1,73 @@
...
           break;
     }
     
     return returnComp;
-
   }
 }
 
+const styles = StyleSheet.create({
+  link: {
```
Comment (AndyCErnst) [inline src/components/Intermediate.js]: this style isn't used in this file

File: src/components/Intermediate.js [hunk 7]
```diff
@@ -1,47 +1,73 @@
...
+    borderWidth: 2,
+    borderColor: 'black',
+
+  },
+  header: {
+    fontSize: 16,
+    margin: 10,
+    padding: 2,
+  },
+  paragraph: {
```
Comment (AndyCErnst) [inline src/components/Intermediate.js]: neither is this one

File: src/components/MyModal.js [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+        style={standardStyle.largeCardIcon}
+      />
+      ) : null;
+    
+    return (
+      <Modal
+        animationType="fade"
+        transparent={true}
+        visible={visible}
+        onRequestClose={() => onRequestClose}>
```
Comment (AndyCErnst) [inline src/components/MyModal.js:32]: I don't see where `onRequestClose` comes from

File: src/components/MyModal.js [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+          <View style={styles.modalViewInner}>
+            <Card
+              style={standardStyle.defaultCard}
+              elevation={4}
+              onPress={() => onPress}
+            >
+              {cardCoverJSX}
+              <Card.Content>
+                <Paragraph>
+                  {loader ? null : "\n" }
```
Comment (AndyCErnst) [inline src/components/MyModal.js:43]: ? why is this needed?

File: src/routes/results.js [hunk 10]
```diff
@@ -1,110 +1,91 @@
...
   render() {
-    // Seth - This is how to pass state between routes with getParam from react-navigation.
     const { navigation } = this.props;
     const response = navigation.getParam('machineLearningResponse', 'NO-response');
-    //const { label, probability } = response.result[0];
-    const label = "Dish Washers";
     const resizedImage = navigation.getParam('resizedImage', 'NO-image');
+    
+    if (!response.result || !response.result[0].label){
+      this.setState({
```
Comment (AndyCErnst) [inline src/routes/results.js:25]: this is a react antipattern, we shouldn't set state in the render method.
It seems like the `response.result` comes in as a route prop which changes what should be displayed, but so does state. So it seems like the route prop should set the initial state in the constructor. That's my recommendation

File: src/routes/landing.js [hunk 11]
```diff
@@ -1,60 +1,47 @@
...
-          <Image source={images.camera}/>
-          <LargeText style={styles.callToAction} text="Autodetect Item"/>
-        </LargeTouchable>
-
-      </ContainerWithNavbar>
+      <View style={standardStyle.pageContainerFixedTop}>
+        <Button
+          style={standardStyle.defaultButton}
+          icon="add-a-photo"
+          mode="contained"
```
Comment (AndyCErnst) [inline src/routes/landing.js:21]: what is mode contained?

**Comment by SethDonohue** [comment]
Sorry Andy just saw your review! I appreciate you looking through this! I'll work on it asap and try to answer all your comments. Some of this code is from Eric's work, so I'll bug him as well.

File: src/routes/landing.js (hunk 11 above)
Comment (SethDonohue) [reply src/routes/landing.js:21]: This is just the style of the button. the docs on these components show all of this.

File: src/components/MyModal.js (hunk 8 above)
Comment (SethDonohue) [reply src/components/MyModal.js:32]: This is just junk filler at the moment because we are not allowing the user to close the modal.
can set it to null as well I believe. Was just following an example.

File: src/components/MyModal.js (hunk 9 above)
Comment (SethDonohue) [reply src/components/MyModal.js:43]: just a quick hack to get a space here when not loader otherwise it looks funny.

File: src/routes/results.js (hunk 10 above)
Comment (SethDonohue) [reply src/routes/results.js:25]: Moved this logic to the componentDidMount lifecycle.

**Comment by SethDonohue** [comment]
Pushed up new code to address issues here as best I could.

File: infoTree/getJSX.js (hunk 2 above)
Comment (SethDonohue) [reply infoTree/getJSX.js]: done

File: src/components/Intermediate.js (hunk 6 above)
Comment (SethDonohue) [reply src/components/Intermediate.js]: done

File: App.js (hunk 1 above)
Comment (SethDonohue) [reply App.js]: done

**Review by AndyCErnst — APPROVED** [review]
thanks for the changes
